pkg/skaffold/util/util.go

```diff
@@ -213,3 +215,36 @@ func BoolPtr(b bool) *bool {
 	o := b
 	return &o
 }
+
+func ReadConfiguration(filename string) ([]byte, error) {
+	switch {
+	case filename == "":
+		return nil, errors.New("filename not specified")
+	case filename == "-":
+		return ioutil.ReadAll(os.Stdin)
+	case strings.HasPrefix(filename, "http://") || strings.HasPrefix(filename, "https://"):
+		return download(filename)
+	default:
+		return ioutil.ReadFile(filename)
+	}
+}
+
+func download(url string) ([]byte, error) {
+	resp, err := http.Get(url)
+	if err != nil {
+		return nil, err
+	}
+	defer resp.Body.Close()
+
+	return ioutil.ReadAll(resp.Body)
+}
+
+func WriteNewConfiguration(config []byte, originalFilename string, overwrite bool) (string, error) {
```

Comment:
can we make this print to stdout instead? Let this function take in an io.Writer from the cmd

Reply:
fixed. moved this logic into the cmd itself, and stuck with using `ioutil.WriteFile` if we're overwriting since you can't overwrite a file using `bufio.NewWriter` without explicitly deleting the file first, which I definitely don't want to do.